app/models/piece.rb

```diff
@@ -128,7 +128,12 @@ def is_obstructed?(x, y)
 
   # Remove the piece from the board
   def remove_piece(piece) 
-    piece.update_attributes(status: :captured)
+    CapturedPiece.create(game_id:    piece.game_id, 
```

Comment:
If I'm understanding this change correctly, any time there is a new `CapturedPiece` created, it will also remove it from the Pieces for the Game instance. Correct? I'm asking because the `in_check?` method I'm currently working on uses the `status` of the pieces to the pieces remaining on the board to see if any of them threatens either King. Before approving, I'd like to double-check that my understanding is correct so the logic of my method doesn't change dramatically given these changes.

Reply:
This is correct. We won't need to use piece status anymore.

Reply:
Sounds good, and makes things a bit easier on my end, too. Thanks!